docs/tutorials/pi/make-a-plant-watering-robot.md

```diff
@@ -0,0 +1,483 @@
+
+Wire the pins as follows:
+
+| MCP3008 ADC Pin | Raspberry Pi Pin |
+| ----------- | ---------------- |
+| VDD | 3.3V |
+| VREF | 3.3V |
+| AGND | GND |
+| DGND | GND |
+| CLK | SCLK |
+| DOUT | MISO |
+| DIN | MOSI |
+| CS/SHDN | GPIO25 |
+
+[**Wiring Diagram**](#wiring-diagram)
+
+### Wire your Soil Moisture Sensor
+
+Next, wire your capacitive soil moisture sensor to your Pi and ADC.
+
+Refer to the following pinout diagram for your capacitive soil moisture sensor:
+
+![Pinout diagram for the capacitive soil moisture sensor.](../../img/plant-watering-pi/moisture-sensor-pinout.png)
+
+Wire the pins as follows:
+
+| Pi | ADC |
+|--|--|
+|<table> <tr><th>Moisture Sensor Pin</th><th>Raspberry Pi Pin</th></tr><tr><td>VCC</td><td>3.3V</td></tr><tr><td>GND</td><td>GND</td></tr> </table>| <table> <tr><th>Moisture Sensor Pin</th><th>MCP3008 ADC Pin</th></tr><tr><td>A0 (Analog Signal Output)</td><td>CH0</td></tr> </table>|
+
+Put the soil moisture sensor inside of your plant box.
+
+[**Wiring Diagram**](#wiring-diagram)
+
+### Wire your Pump
+
+Now, wire and power your Peristaltic Pump [motor](/components/motor/) to complete your hardware setup.
+
+1. Connect your battery to the DC power pins on your motor speed controller.
+2. Then, connect the output pins on your motor speed controller to the pump.
+3. Connect the GND pin on the pump to the breadboard and terminal.
+4. Connect the PWM pin on the pump to [Pin 12](https://pinout.xyz/pinout/pin12_gpio18) of the Pi.
+5. Connect the plastic tubing to the pump. Put the suction end inside of your water box, and the output end inside of your plant box.
+
+### Wiring Diagram
+
+Refer to the following full wiring diagram for your hardware setup:
+
+![The full wiring diagram for all the hardware for the Plant Watering Robot.](../../img/plant-watering-pi/full-wiring.png)
+
+## Program Your Plant Watering Robot
+
+### Enable SPI on your Pi
+
+Now that you have wired your ADC and Moisture Sensor, make sure that SPI is enabled on your Pi.
+
+SSH into your Pi and run the following command:
+
+``` shell
+sudo raspi-config
+```
+
+Once the `raspi-config` interface is open, navigate to **Interface Options**:
+
+![Raspi-config Tool interface with Interface Options selected.](../../img/plant-watering-pi/interface-options.png)
+
+Then, click on **SPI**:
+
+![Raspi-config Tool interface with SPI selected.](../../img/plant-watering-pi/spi.png)
+
+Now, click **Yes** to enable SPI:
+
+![Raspi-config Tool interface with Yes selected for SPI enablement.](../../img/plant-watering-pi/spi-enabled.png)
+
+Finally, click **Finish** and restart your Pi.
+
+### Test your Soil Moisture readings on your Pi
+
+Next, install the Adafruit ADC library [`Adafruit_CircuitPython_MCP3xxx`](https://github.com/adafruit/Adafruit_CircuitPython_MCP3xxx/) on your Pi.
+
+First, make sure you have installed the required dependencies:
```

Comment:
This is the first time you mention them so just instruct them to install them instead of saying make sure you have them installed